charts/astronomer/templates/houston/worker/houston-worker-deployment.yaml

```diff
@@ -102,11 +102,6 @@ spec:
               value: "INFO"
             - name: GRPC_TRACE
               value: "all"
-            - name: DATABASE_URL
-              valueFrom:
-                secretKeyRef:
-                  name: {{ template "houston.backendSecret" . }}
```

Comment:
oh, it's another secret, @pgvishnuram please double check it.

Reply:
same key cannot be used, the same is actually in helper templates too

Reply:
https://github.com/astronomer/astronomer/blob/091dd67965b316d8aff4c57ef355bcaee1ff4a58/charts/astronomer/templates/_helpers.yaml#L203-L207

yeah looks ok. My point it's same env variable but with different values, potentially it can cause issues in prod.